experimental/IntersectionTheory/src/Main.jl

```diff
@@ -248,74 +278,92 @@ line_bundle(X::AbstractVariety, D::MPolyDecRingElem) = AbstractBundle(X, 1, 1+D[
 
 @doc raw"""
     degree(X::AbstractVariety)
-Compute the degree of $X$ with respect to its polarization."""
+
+Compute the degree of $X$ with respect to its polarization.
+"""
 degree(X::AbstractVariety) = integral(X.O1^X.dim)
 
 @doc raw"""
     tangent_bundle(X::AbstractVariety)
     tangent_bundle(X::TnVariety)
+
 Return the tangent bundle of a abstract_variety $X$. Same as `X.T`.
 """
 tangent_bundle(X::AbstractVariety) = X.T
 
 @doc raw"""
     cotangent_bundle(X::AbstractVariety)
     cotangent_bundle(X::TnVariety)
+
 Return the cotangent bundle of a abstract_variety $X$.
 """
 cotangent_bundle(X::AbstractVariety) = dual(X.T)
 
 @doc raw"""
     canonical_class(X::AbstractVariety)
-Return the canonical class of a abstract_variety $X$."""
+
+Return the canonical class of a abstract_variety $X$.
+"""
 canonical_class(X::AbstractVariety) = -chern_class(X.T, 1)
 
 @doc raw"""
     canonical_bundle(X::AbstractVariety)
-Return the canonical bundle of a abstract_variety $X$."""
+
+Return the canonical bundle of a abstract_variety $X$.
+"""
 canonical_bundle(X::AbstractVariety) = det(cotangent_bundle(X))
 
 @doc raw"""
     bundles(X::AbstractVariety)
     bundles(X::TnVariety)
+
 Return the tautological bundles of a abstract_variety $X$. Same as `X.bundles`.
 """
 bundles(X::AbstractVariety) = X.bundles
 
 @doc raw"""
     total_chern_class(X::AbstractVariety)
     total_chern_class(X::TnVariety)
+
 Compute the total Chern class of the tangent bundle of $X$.
 """
 total_chern_class(X::AbstractVariety) = total_chern_class(X.T)
 
 @doc raw"""
     chern_class(X::AbstractVariety, k::Int)
     chern_class(X::TnVariety, k::Int)
-Compute the $k$-th Chern class of the tangent bundle of $X$.
+C
+ompute the $k$-th Chern class of the tangent bundle of $X$.
```

Comment:
seems to be an unwanted change...

Reply:
Indeed, ouch, thanks!
```suggestion

Compute the $k$-th Chern class of the tangent bundle of $X$.
```